wp-content/themes/amplify/less/amplify-embed-feed.less

```diff
@@ -0,0 +1,104 @@
+.amplify-embed-feed {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    padding: 0 20px;
+    .horizontal & {
+        @media (min-width: 1000px) {
+            flex-direction: row;
+            position: relative;
+            .stories-title {
+                position: absolute;
+                top: @measure;
+            }
+            .tag-header {
+                margin-top: @measure + 5rem
+            }
+            .saved-links {
+                display: flex;
+                flex-wrap: wrap;
+                max-width: 70%;
+                margin-top: @measure + 5rem;
+                .saved-link {
+                    flex-grow: 1;
+                    flex: 1 1 20%;
+                    margin: @measure + 1rem 10px;
+                    position: relative;
+                    &:first-of-type {
+                        margin-top: @measure + 1rem;
+                    }
+                }
+            }
+        }
+    }
+    .tag-header {
+        display: flex;
+        align-items: center;
+        margin: @measure 0;
+        img {
+            float: left;
+            max-width: 40%;
+        }
+        .inner-tag-header {
+            margin-left: @measure;
+            .tag-title {
+                margin-bottom: @measure;
+            }
+            .tag-description {
+                display: none;
+            }
+        }
+    }
+    .stories-title {
+        margin-bottom: @measure;
+    }
+    .hide-header {
+        display: none;
+    }
+    .saved-links {
+        &.rows-2 {
+            grid-template-columns: repeat(2, 1fr);
+        }
+        &.rows-3 {
+            grid-template-columns: repeat(3, 1fr);
+        }
+        &.rows-2,
+        &.rows-3 {
+            display: grid;
+            .saved-link {
+                &:first-of-type {
+                    margin-top: 10px;
+                }
+                margin: 10px;
+            }
+            @media (max-width: 767px) {
+                display: block;
+            }
+        }
+        .saved-link {
+            border: 2px solid @darkblue;
+            margin: @measure + 1rem auto;
+            padding: 20px;
+            &:first-of-type {
+                margin-top: 0;
+            }
+            &:not(.show-link-description) {
+                .description {
+                    * {
+                        &:not(.category-btn) {
+                            display: none;
+                        }
+                    }
+                }
+            }
```

Comment:
This doesn't seem to be working for me; with `&show_link_descriptions=true` and without it the saved link descriptions appear. Every screenshot in this review was taken without it, and the descriptions still appeared.

It might be because it's hiding children of `.description` and not `.description` itself, and the element on the page is a div containing a text node, rather than any child elements.

Reply:
@benlk Did you have any other params present when testing this?

It seems to work fine for me. (Note only the 2nd link in the screenshots has a desc.)

Without any param:
<img width="1391" alt="Screen Shot 2019-12-16 at 11 56 09 AM" src="https://user-images.githubusercontent.com/18353636/70926576-5c0f9980-1ffb-11ea-99c9-e0d0c81adbb6.png">

With `show_link_descriptions=false`:
<img width="1393" alt="Screen Shot 2019-12-16 at 11 57 00 AM" src="https://user-images.githubusercontent.com/18353636/70926570-5b770300-1ffb-11ea-8248-35e692c4bb15.png">


With `show_link_descriptions=true`:
<img width="1391" alt="Screen Shot 2019-12-16 at 11 56 28 AM" src="https://user-images.githubusercontent.com/18353636/70926572-5b770300-1ffb-11ea-963b-df89f6640481.png">

Reply:
Even at https://amplify.inndev.test/?amplify-feed the descriptions aren't hidden.

Here's the markup of an affected link:

```html
<div class="saved-link clearfix ">
                <h5><a href="https://www.bettergov.org/news/lightfoot-s-actions-on-environmental-campaign-promises-are-off-to-a-slow-start/" target="_blank">Lightfoot’s Actions on Environmental Campaign Promises Are Off to a Slow Start</a>                </h5>

                <p class="source">By: <span><a href="https://www.bettergov.org/news/lightfoot-s-actions-on-environmental-campaign-promises-are-off-to-a-slow-start/" target="_blank">Better Government Association</a></span></p><div class="description">Chicago mayor says budget constraints have cooled her ambitious plans. Her 2020 budget includes no new funding for additional pollution inspectors.</div>            </div>
```

The description content contains no HTML tags; it was just plain text pasted into the link roundup editor.

If we want to be sure that there are paragraph tags there, would running the description through https://developer.wordpress.org/reference/functions/wpautop/ work?

Reply:
> The description content contains no HTML tags; it was just plain text pasted into the link roundup editor.

@benlk Ahhhhhh, that's what the issue was. Yes, running it through `wpautop` fixes it.

Updated in https://github.com/INN/umbrella-inndev/pull/131/commits/5e361a3c1179764f66466b83d817ef0289b7c9e5.